summerofcode/2022.md

```diff
@@ -405,6 +405,14 @@ Description: The [in-toto Jenkins plugin](https://github.com/jenkinsci/in-toto-p
 - Mentor(s): Sedef Savas (@sedefsavas) Richard Case (@richcase)
 - Upstream Issue (URL): https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/2178
 
+### Prometheus
+
+#### Add Run-Length-Encoding for samples in TSDB
+
+- Description: Prometheus stores the scraped data in it's own [Time Series Database](https://www.wikiwand.com/en/Time_series_database) (TSDB). Due to the nature of often collecting simliar values they can be stored in sequenence using optimized encodings. In Prometheus v2.0 and above all values are encoded using [XOR'ed](https://blog.acolyer.org/2016/05/03/gorilla-a-fast-scalable-in-memory-time-series-database/) int64. Almost since the initial v2.0 release there is a long standing issue on adding other encodings to Prometheus' TSDB. Especially cases where either all values of the series are either `0` or `1` are prime candidates for [Run-Length-Encoding](https://www.wikiwand.com/en/Run-length_encoding) (RLE). After conducting an [experiment on the real world feasbility for adding RLE](https://github.com/prometheus/prometheus/pull/10275), that looks very promising (saving up to 10% disk), we want to work with you to implement RLE in TSDB and improve Prometheus' storage!
```

Comment:
```suggestion
- Description: Prometheus stores the scraped data in it's own [Time Series Database](https://en.wikipedia.org/wiki/Time_series_database) (TSDB). Due to the nature of often collecting simliar values they can be stored in sequenence using optimized encodings. In Prometheus v2.0 and above all values are encoded using [XOR'ed](https://blog.acolyer.org/2016/05/03/gorilla-a-fast-scalable-in-memory-time-series-database/) int64. Almost since the initial v2.0 release there is a long standing issue on adding other encodings to Prometheus' TSDB. Especially cases where either all values of the series are either `0` or `1` are prime candidates for [Run-Length-Encoding](https://en.wikipedia.org/wiki/Run-length_encoding) (RLE). After conducting an [experiment on the real world feasbility for adding RLE](https://github.com/prometheus/prometheus/pull/10275), that looks very promising (saving up to 10% disk), we want to work with you to implement RLE in TSDB and improve Prometheus' storage!
```

Reply:
Seems like I slipped some wikiwand rather than wikipedia :upside_down_face: